build.sbt

```diff
@@ -4,15 +4,15 @@ name := """codacy-engine-pylint"""
 
 version := "1.0-SNAPSHOT"
 
-val languageVersion = "2.11.12"
+val languageVersion = "2.12.7"
 
 scalaVersion := languageVersion
 
 resolvers := Seq("Sonatype OSS Snapshots".at("https://oss.sonatype.org/content/repositories/releases")) ++ resolvers.value
 
 libraryDependencies ++= Seq(
-  "com.typesafe.play" %% "play-json" % "2.4.8",
-  "com.codacy" %% "codacy-engine-scala-seed" % "2.7.8"
+  "com.typesafe.play" %% "play-json" % "2.7.3",
```

Comment:
You should remove play dependency everywhere since it already is provided by the seed.
https://github.com/codacy/codacy-engine-scala-seed/blob/master/project/Dependencies.scala#L5

Please always check the dependency graph before doing the type of changes or we will end up with multiple evicted stuff in our classpaths.